packages/flutter/test/material/about_test.dart

```diff
@@ -449,6 +449,25 @@ void main() {
     expect(rootObserver.dialogCount, 0);
     expect(nestedObserver.dialogCount, 1);
   });
+
+  testWidgets(
```

Comment:
Make this all one line:
```
  testWidgets("AboutListTile's child should not be offset when the icon is not specified.", (WidgetTester tester) async {
```

Reply:
Yeah I don't like it either but it's a formatter issue. The problem is this file is hard to work on because you always have to avoid formatting the whole file. What do you think?

Reply:
We typically don’t work on Flutter source code with dartfmt turned on. I’d just try to stick to Flutter’s style guide thats in the wiki and turn the formatter off

Reply:
I guess I missed that. I'll do that. Thank you